Tetrahedral_remeshing/include/CGAL/Tetrahedral_remeshing/internal/peel_slivers.h

```diff
@@ -70,16 +72,16 @@ std::size_t peel_slivers(C3T3& c3t3,
     Cell_handle c = c_i.first;
     const std::array<bool, 4>& f_on_surface = c_i.second;
 
-    boost::optional<Surface_patch_index> patch;
+    std::optional<Surface_patch_index> patch;
     for (int i = 0; i < 4; ++i)
     {
       if (f_on_surface[i])
       {
         Surface_patch_index spi = c3t3.surface_patch_index(c, i);
-        if (patch != boost::none && patch.get() != spi)
+        if (patch.has_value() && patch.value() != spi)
         {
           //there are 2 different patches
-          patch = boost::none;
+          patch = std::nullopt;
```

Comment:
```suggestion
          patch.reset();
```

This can be replaced with ``patch.reset()`` along with other similar assignments to ``std::nullopt``. ``std::optional<T>::reset`` is a better alternative to clear a value stored in a std::optional.